libmscore/fermata.cpp

```diff
@@ -110,6 +110,7 @@ void Fermata::write(XmlWriter& xml) const
       xml.tag("subtype", Sym::id2name(_symId));
       writeProperty(xml, Pid::TIME_STRETCH);
       writeProperty(xml, Pid::PLAY);
+      writeProperty(xml, Pid::MIN_DISTANCE);
       if (!isStyled(Pid::OFFSET))
             writeProperty(xml, Pid::OFFSET);
```

Comment:
@MarcSabatella Do you mean `writeProperty(xml, Pid::MIN_DISTANCE)` should be placed under `if (!isStyled(Pid::OFFSET))`?

Reply:
So no change needed?